src/components/map/layers/edit/EditNodeLayer.tsx

```diff
@@ -34,12 +40,6 @@ class EditNetworkLayer extends Component<IEditNetworkLayerProps> {
         );
     }
 
-    private onMoveMarker = (node: INode) =>
```

Comment:
onMoveMarker is required to update the store with new coordinate values of the node

Reply:
Good point